src/fee-accounts/services/fee-accounts.service.ts

```diff
@@ -1,30 +1,30 @@
-import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
 import { Observable, of } from 'rxjs';
 import {map} from '../../../node_modules/rxjs/operators';
-import {Payment} from '../models/pba-transactions';
+import {SingleAccontSummaryMock} from '../mock/singleAccontSummary.mock';
```

Comment:
@SivaprakasamSivaprakasam - can you refactor this to correct the spelling mistake, i.e. `SingleAccountSummaryMock` and `singleAccountSummary.mock`